task.sql

```diff
@@ -3,9 +1,25 @@
+USE ShopDB;
 
--- Some data should be created outside the transaction (here)
+INSERT INTO Products (ID, Name, Description, Price, WarehouseAmount)
+VALUES (1, 'AwersomeProduct', 'An awesome product!', 100, 50);
```

Comment:
The description 'An awesome product!' is correct, but ensure it matches any predefined descriptions if applicable.